### ISSUE TITLE
Need to pass only args to evaluate callback

### ISSUE THREAD

**Issue by combinesoma** [getgauge/taiko#2687]
Hi Team - In the current version of the evaluate API, there is no provision to send only args to the callback, we need to pass an element along with the args param, is there any way where we can pass only arguments to the callback function?

Example: 
await evaluate( (args) => { }, options)

**Comment by mzissa** [comment]
I agree with you. I faced this issue as well. The way I got around it is by passing a dummy select like this:

```
const dummyDiv = $("div");
await evaluate(dummyDiv, function (x) { });
```
